xml/depl_vmware.xml

```diff
@@ -174,7 +174,15 @@
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>VLAN Interface</term>
+    <term><guimenu>Datastore Names</guimenu></term>
```

Comment:
Regex to match the name of a datastore.

This setting specifies the data stores to use with Nova Compute. For example, value  "nas.*" selects all the data stores that have a name starting with "nas". If this line is omitted, Nova Compute uses the first data store returned by the vSphere API. It is recommended not to use this field and instead remove data stores that are not intended for OpenStack.